src/components/basin-analyzer/table/constants.js

```diff
@@ -0,0 +1,74 @@
+import { LEGENDS, SHORT_NAMES } from '../../map/constants';
+
+const renderStatic = (indicator, value, fixed = 0) => `${value && value.toFixed(fixed)}${(LEGENDS[indicator] && LEGENDS[indicator].unit) || ''}`;
+const renderPercent = (indicator, value, fixed = 0) => `${value && (value * 100).toFixed(fixed)}%`;
+export const INDICATOR_RAW_VALUE_RENDER_MAP = {
+  bws_cat: value => renderPercent('bws_cat', value),
+  bwd_cat: value => renderPercent('bwd_cat', value),
+  gtd_cat: value => renderStatic('gtd_cat', value, 2),
+  cep_cat: value => renderStatic('cep_cat', value, 2),
+  udw_cat: value => renderPercent('udw_cat', value),
+  usa_cat: value => renderPercent('usa_cat', value)
+};
+
+export const COLUMNS = [
+  {
+    label: 'Watershed ID',
+    value: 'watershed_id'
+  },
+  {
+    label: 'Name',
+    value: 'location_name'
+  },
```

Comment:
This seems like a lot of duplication of `src/components/basin-analyzer/constants.js`, but I trust your judgement if you did that on purpose.